thrust_manager:
  auv_ns: auh
  control_mode: pid
  thruster_count: 6

  thrust_matrix: [
     1,  1,  0,  0,  0,  0,
    -1,  1,  0,  0,  0,  0,
     0,  0,  1,  1,  1,  1,
     0,  0,  1, -1,  1, -1,
     0,  0, -1, -1,  1,  1
  ]

  buoyancy_compensation: -20
  vertical_thrusters: [2, 3, 4, 5]